simglucose/simulation/env.py

```diff
@@ -34,7 +36,58 @@ def risk_diff(BG_last_hour):
 
 
 class T1DSimEnv(object):
+    '''
+    Observation:
+        Type: Box(4)
+        Num     Observation                 Min         Max
+        0       BG                          69.         351.
+        1       Insulin                     0.0         Inf
+        2       CHO                         0.0         Inf
```

Comment:
CHO is not in Observation space. It is recorded in `info` dictionary. It is an additional information, and it is optional for agent to leverage. Strictly, agent is required to do a decent job without meal information, but it is even better if it can utilize the meal information to enhance the glucose control.